src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -58,11 +54,18 @@ export const proxyRequest = ({
   timeout,
   payload,
   rejectUnauthorized,
+  path,
 }: Args) => {
-  const { hostname, port, protocol, pathname, search } = uri;
+  const { hostname, port, protocol, search } = uri;
   const client = uri.protocol === 'https:' ? https : http;
-  const encodedPath = encodePathname(pathname);
+  let pathname = uri.pathname;
   let resolved = false;
+  // Compare original request path to percent-encoded path returned by node URL method
+  const requiresEncoding = trimStart(path, '/') !== trimStart(pathname, '/');
```

Comment:
This solution seems specific to the the `/` character. Could there be other cases where the pathname requires encoding? What about other characters covered here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Reply:
Ah ok sorry I didn't get this on first read through for some reason. This LGTM